solution-jivanov/lib/model.js

```diff
@@ -0,0 +1,31 @@
+// model.js
+// Model class - base model class
+// contains functionality that is/can be common for all models
+
+const Base = require('./base.js');
+
+const AWS = require('aws-sdk');
+
+let dynamoDb;
+
+class Model extends Base{

```

Comment:
I've noticed you took a relatively abstracted OOP approach for the lambda code to leverage.

How do you come to that design decision and what were the alternatives you considered?

Reply:
Oh, yes, as mentioned above - just trying to showcase understanding of more stuff (abstraction, classes, inheritance, etc.). Not really needed here.

The good thing about microservices (compared to a monolith) is that we can make them as simple (or as complex) as needed, completely independently from the other microservices. If a service can benefit from that level of abstraction - then sure, let's use it. If it does not need it - no need to have it either. "KISS" in action :)

For that particular service - we certainly do not need it. A simple service and a model classes would have been enough.